.github/workflows/test.yaml

```diff
@@ -30,5 +30,5 @@ jobs:
       - name: Install and test
         run: |
           cd $GITHUB_WORKSPACE
-          nix develop --command bash -c "cargo install --path . && sh run_test.sh"
+          nix develop --override-input "versions/scaffolding" . --command bash run_test.sh
```

Comment:
@robbiecarlton i'd like you to comment on this change specifically. instead of building scaffolding with `cargo` it changes the revision that nix will use to build the holonix-based devShell.